frontend.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+/*
+ * Gets a database object to use for database queries. This database's fetch mode is set to FETCH_ASSOC by default.
+ *
+ * @Return returns the object to use for our given queries.
+ */
+function getDBConnection() {
```

Comment:
You already have a database connection function in cms_functions.  Consider using just one connection function.

Reply:
I wanted to keep the cms functions and the frontend functions seperate. Eventually I just settled for extracting the code out of the function to use once in index.php